data/ui/book_detail.blp

```diff
@@ -243,11 +201,35 @@ template $BookDetail: Adw.NavigationPage {
 }
 
 menu book_menu_model {
+  section {
+    item {
+      action: 'app.mark_book_as_read';
+      label: _("Mark as Read");
+    }
+
+    item {
+      action: 'app.reset_book';
+      label: _("Remove from Recents");
+    }
+
```

Comment:
This action doesn't belong in here. In fact, I think it should be hidden even on the "Author" and "Reader" pages, as it's related only to the "Recents" page